lib/Exporter/ONNXModelWriter.cpp

```diff
@@ -211,51 +372,55 @@ ONNXModelWriter::ONNXModelWriter(const std::string &modelFilename, Function &F,
     opsetImportProto->set_version(opsetVersion);
     auto *graphProto = modelProto.mutable_graph();
     graphProto->set_name("glow");
-
     // Use post order graph traversal.
     // If node is constant with uses, turned into "input" and create a tensor
     // If node is placeholder with uses, turned into "input" with tensor shape,
     // except the case when placeholder has use as SaveNode.
     // Otherwise call common operators method or special operators and write
     // protobuf inputs from node inputs and protobuf outputs from uses.
-    GraphPostOrderVisitor visitor(G_);
+    GraphBackwardsPostOrderVisitor visitor(G_);
```

Comment:
Why do we want pre order instead of post order?

Reply:
We want post order but with a modified visiting order between current node and its inputs.